retrofit-converters/jaxb/src/test/resources/META-INF/services/javax.xml.bind.JAXBContext

```diff
@@ -0,0 +1 @@
+org.eclipse.persistence.jaxb.JAXBContextFactory
```

Comment:
What does this do?

Reply:
By this file I changed an implementation of JAXBContextFactory to EclipseLink's one, because it supports both XML & JSON, so I was able to write tests for both formats.

https://docs.oracle.com/javase/9/docs/api/java/util/ServiceLoader.html